fvm/script.go

```diff
@@ -198,11 +198,13 @@ func (executor *scriptExecutor) executeScript() error {
 			Source:    executor.proc.Script,
 			Arguments: executor.proc.Arguments,
 		},
-		common.ScriptLocation(executor.proc.ID))
+		common.ScriptLocation(executor.proc.ID),
+	)
+	populateErr := executor.output.PopulateEnvironmentValues(executor.env)
 	if err != nil {
 		return err
```

Comment:
```suggestion
		return multierror.Append(err, populateErr)
```

so we don't accidentally loose the `populateErr` if both errors happen.

Reply:
Cool, didn't know about this :bow: 
Updated in https://github.com/onflow/flow-go/pull/4572/commits/574fb22f64c8dd2e60dc957bb3f8883f89195833